SpatialConstAffine.lua

```diff
@@ -0,0 +1,39 @@
+local SpatialConstAffine, parent = torch.class('inn.SpatialConstAffine', 'nn.Module')
+
+function SpatialConstAffine:__init(nOutputPlane, inplace)
+  parent.__init(self)
+  self.a = torch.Tensor(1,nOutputPlane,1,1)
```

Comment:
right, it's not garbage.
but I still think that with minimal changes this could be way more generic. But let's leave it like that for the moment.